Apps/Src/ReadTritium.c

```diff
@@ -118,7 +95,7 @@ void Task_ReadTritium(void *p_arg){
 	}
 }
 
-void MotorController_Restart(void){
+static void restartMotorController(void){
 	CANDATA_t resetmsg = {0};
 	resetmsg.ID = MOTOR_RESET;
 	CANbus_Send(resetmsg, true, MOTORCAN);
```

Comment:
Ah, it doesn't. The way to do this would apparently be to see if we receive messages after we send the restart command. This PR is blocking another one, though, so I'll save these notes to implement in M3/after other stuff is merged in. Thank you for leaving a review!